sdks/wasm/DebuggerTestSuite/Support.cs

```diff
@@ -790,6 +790,12 @@ internal async Task<Result> SetBreakpoint (string url_key, int line, int column,
 			return bp1_res;
 		}
 
+		internal async Task<Result> SetPauseOnException (string state)
```

Comment:
This is fine too, but I meant that keep the enum here too, but convert that enum to a string in this method, so `library_mono.js` gets a string.

Reply:
library_mono.js is already getting the string why should we keep the enum in the Support.cs?

Reply:
Oh, this is `Support.cs`. ignore me 😬